modules/dashboard/README.md

```diff
@@ -79,8 +79,10 @@ module "this" {
 | <a name="module_alb_ingress_target_response_time_widget"></a> [alb\_ingress\_target\_response\_time\_widget](#module\_alb\_ingress\_target\_response\_time\_widget) | ./modules/widgets/alb_ingress/target_response_time | n/a |
```

Comment:
please have tests/example on how to use the new 2 dashboard block in /tests/ folder